lib.php

```diff
@@ -430,9 +457,13 @@ function local_augmented_teacher_send_activity_recommendation() {
              WHERE rem.messagetype = ?
                AND rem.enabled = 1
                AND rem.deleted = 0";
+    $sql_params = array(0, 1, MESAGE_TYPE_REMINDER);
 
-    $rs = $DB->get_recordset_sql($sql, array(MESAGE_TYPE_RECOMMEND));
+    $rs = $DB->get_recordset_sql($sql, $sql_params);
```

Comment:
This should be another repo for getting the reminders.

The $rs variable doesn't say anything about what kind of data it contains.

Do something like this:
$reminders = $reminders_repo->get_all();